migrate/test_repo_checks.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import MagicMock, call
+
+import pytest
+
+from .repo_checks import EnsureLabels
+
+
+@pytest.fixture
+def maintenance_label():
+    maintenance_label = MagicMock()
+    maintenance_label.name = ":hammer_and_wrench: maintenance"
+    maintenance_label.color = "169509"
+
+    return maintenance_label
+
+
+class TestEnsureLabels:
```

Comment:
Could you add some comments with context on some of the assert statements that just check that a value is true?